### ISSUE TITLE
[bench request]: Add Quire (Qwen3.5-4B frozen, option-letter readout over two option orders, TypeSafe wire format)

### ISSUE THREAD

**Issue by silentfishwm** [fstandhartinger/jevbench#45]
Hi Florian, I'd like to submit **Quire** to JevBench.

Quire reads typed decisions from a frozen language model: one forward pass per option ordering, the distribution taken from the option-letter logits at the first assistant token, with no generated tokens. It serves TypeSafe's wire format (`POST /v1/systemone`), so your unchanged `typesafe` adapter runs it.

## System details

- **Name:** Quire (Qwen3.5-4B, frozen)
- **Repo:** https://github.com/aflowx/quire, tag `v0.1.0` (commit `7b44d8c`), MIT
- **Weights:** `Qwen/Qwen3.5-4B`, BF16, revision `851bf6e806efd8d0a36b00ddf55e13ccb7b8cd0a`, unmodified: no adapter, no training, no calibration file (temperature 1)
- **Readout:** each question is read with its options in two cyclic orders and the two distributions are averaged. The state is prefilled once, and both orderings run as one batch over the cached prefix.
- **Prompt:** Quire's own, fixed for every request: a system instruction that frames the question as a test run against the state, then `<state>`, `<test>` and lettered `<options>`, thinking off. `noul` is read over true/false options, `score` over its levels.
- **Output tokens:** 0 per decision
- **Choices over 26 options** are answered by a per-option yes/no pass in the same request. No JevBench item has that many options, so this path never runs here.

## How to run

One GPU with about 10 GB (measured on an NVIDIA L40, BF16). `flash-linear-attention` is required for the Gated DeltaNet kernels and is installed by the `torch` extra. `causal-conv1d` is optional and measured no faster.

```sh
git clone https://github.com/aflowx/quire && cd quire && git checkout v0.1.0
pip install -e ".[torch]"
quire-serve --backend torch --model Qwen/Qwen3.5-4B --revision 851bf6e806efd8d0a36b00ddf55e13ccb7b8cd0a \
    --host 127.0.0.1 --port 8778
# the unchanged typesafe adapter, no key:
python -m jevbench.cli run --tasks <tasks> --adapter typesafe --endpoint http://127.0.0.1:8778 --key-env '' --model quire ...
```

It loads the model and answers one warm-up request before printing `quire ready`. There is no authentication, so bind it to loopback.

## Reference local run (public items)

Run through your harness at `f79a1ca` with `--adapter typesafe` against the server above, on one L40, serial:

| tier | items | Quire |
|---|---|---|
| easy | 48 | 1.000 |
| standard (original) | 72 | 0.903 |
| hard | 111 | 0.658 |

Every request returned a valid distribution (231 of 231). Hard-tier top-label ECE 0.090, fidelity on the 10 public probability items 0.657, p50 latency 122 ms raw on the standard tier. The endpoint reports `usage.input_tokens` as the state once plus both orderings' suffixes (mean 789 per decision), which is what it computes. Per-item outputs are in `results/jevbench/harness-typesafe-adapter/` in the repo. These are our numbers on our hardware, not a claim about the held-out items or your GPU.

**Cost basis:** the Qwen3.5-4B hosted list price you used for the other frozen 4B rows, $0.03 per million input tokens.

## Disclosure

- No training of any kind. The weights are the published checkpoint, and the answers come from a fixed prompt and an average over two option orders.
- The public items were used as a development gate. We compared configurations on them (two prompt styles, one vs. two orderings, and two LoRA adapters that were rejected and are not part of this system), and a serving bug with JSON-object states was found by running your harness on them. The prompt wording itself was chosen among three candidates on our own held-out synthetic items and then checked on the public items.
- No JevBench text appears in any prompt, template or data the system uses.

Thanks for running the benchmark.

**Comment by fstandhartinger** [comment]
Thanks — this is logged for reachability checks and the JevBench measurement queue. We’ll only make benchmark claims after our own run.

- Harold

**Comment by silentfishwm** [comment]
Update: please run tag **`v0.1.1`** instead of `v0.1.0`. Same weights, prompt and answers. It adds one temperature per answer type (choice 1.40, noul 1.70, score 1.40), fitted only on our own held-out items, never on JevBench items. It's on by default in `quire-serve`, so the run command is unchanged apart from `git checkout v0.1.1`.

On the public hard tier with your harness: ECE 0.090 → 0.061, and no answer changes. Details are in `docs/RESULTS.md`.
